src/components/capture/Details.tsx

```diff
@@ -46,11 +82,106 @@ function CaptureDetails({ lastPubId, disableLogs }: Props) {
+                {shardDetails && (
+                    <>
+                        {shardDetails.errors && (
+                            <Grid item xs={12}>
+                                <Alert
+                                    severity="error"
+                                    sx={{
+                                        '& .MuiAlert-message': {
+                                            width: '100%',
+                                        },
+                                    }}
+                                >
+                                    <Accordion>
+                                        <AccordionSummary
+                                            expandIcon={<ExpandMoreIcon />}
+                                        >
+                                            <Typography>
+                                                <FormattedMessage id="captureDetails.shardDetails.errorTitle" />
+                                            </Typography>
+                                        </AccordionSummary>
+
+                                        <AccordionDetails>
+                                            <Box sx={{ height: 250 }}>
+                                                <Editor
+                                                    defaultLanguage=""
+                                                    theme={
+                                                        theme.palette.mode ===
+                                                        'light'
+                                                            ? 'vs'
+                                                            : 'vs-dark'
+                                                    }
+                                                    options={{
+                                                        lineNumbers: 'off',
+                                                        readOnly: true,
+                                                        scrollBeyondLastLine:
+                                                            false,
+                                                        minimap: {
+                                                            enabled: false,
+                                                        },
+                                                    }}
+                                                    onMount={handlers.mount}
+                                                    value={shardDetails.errors
+                                                        .join(NEW_LINE)
+                                                        .split(/\\n/)
+                                                        .join(NEW_LINE)
+                                                        .replaceAll(
+                                                            /\\"/g,
+                                                            '"'
+                                                        )}
```

Comment:
Since this is done else where we should probably make this a service and share this.